### PR TITLE
Story/sus 1

### PR DESCRIPTION
**Pull request by namrathaov**
Concepts page has a link to edit configuration.
Created a new JSP page to edit configuration with checkboxes.

### CODE REVIEW

File: sustainability/src/main/java/edu/asu/diging/sustainability/web/ListConceptController.java [hunk 1]
```diff
@@ -40,19 +41,31 @@ public String list(Model model) {
...
             for (IConcept child : concept.getChildren()) {
                 addNode(child, children, mapper);
             }
-            
+
         }
     }
+
+    @RequestMapping(value = "/admin/concept/configuration")
+    public String configuration(Model model) {
```
Comment (jdamerow) [inline sustainability/src/main/java/edu/asu/diging/sustainability/web/ListConceptController.java:61]: needs to have its own controller (roughly one controller per page)

File: sustainability/src/main/java/edu/asu/diging/sustainability/web/ListConceptController.java [hunk 2]
```diff
@@ -40,19 +41,31 @@ public String list(Model model) {
...
+        List<IConcept> concepts = conceptManager.getTopConcepts();
+        model.addAttribute("concepts", concepts);
```
Comment (jdamerow) [inline sustainability/src/main/java/edu/asu/diging/sustainability/web/ListConceptController.java:63]: merge with line 62

File: sustainability/src/main/java/edu/asu/diging/sustainability/web/ListConceptController.java [hunk 3]
```diff
@@ -40,19 +41,31 @@ public String list(Model model) {
...
+        List<String> roles = new ArrayList<String>();
+        roles.add("Researcher");
+        roles.add("Practitioner");
```
Comment (jdamerow) [inline sustainability/src/main/java/edu/asu/diging/sustainability/web/ListConceptController.java:66]: these need to come from somewhere configurable, not hardcoded.

File: sustainability/src/main/webapp/WEB-INF/views/admin/concepts/configuration.jsp [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    </div>
+    
+    <div>
+      <c:forEach items="${concept.children}" var="child">
+        <div class="panel-body"">
+	      <li class="list-group-item">
+	        ${child.name}
+	        <c:forEach items="${roles}" var="role">
+	          <label class="switch checkbox-switch">
+	            <input type="checkbox" class="default" name="${role}" value="${role}"> ${role}
```
Comment (jdamerow) [inline sustainability/src/main/webapp/WEB-INF/views/admin/concepts/configuration.jsp:23]: name and value should be different (value can change over time as stakeholders change their mind, values need to stay the same in most cases for backwards compatibility).

**Review by jdamerow — CHANGES_REQUESTED** [review]
where is the configuration stored?

**Comment by namrathaov** [comment]
I mistook that this ticket was only for designing the UI of the page. I will work on storing it.

**Pull request closed without merging**